windows/src/desktop/kmshell/xml/config.css

```diff
@@ -242,78 +287,83 @@ html
   font-weight: bold;
   white-space: nowrap;
 }
+
+.kbd_list_item .list_icon
+{
+  margin: 0px 6px 0px 2px;
+}
+
 .list_icon
 {
   display: inline;
   margin: 2px 6px 0px 2px;
   vertical-align: middle;
 }
+
 .list_detail
 {
-  background: #f0f0ff;
+
+  background: #ffffff;
   clear: both;
   font-size: 11px;
-  margin: 1px;
   display: none;
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
 }
 
-.list_detail > table > tbody > tr > td
+.checkbox_hidden
 {
-  border: solid 1px #79c3da;
-  vertical-align: top;
+  visibility: hidden;
+  opacity: 0;
+  display: none;
 }
 
-.list_item.expanded .list_expand
-{
-  background-image: url('collapse20.png');
+input[type=checkbox]+label {
+  background: #dadddc;
+  border-radius: 4px;
+  color: #33333f;
+  border: none;
+  vertical-align: top;
+  font-size: 12px;
+  height: 25px;
+  margin-right: 8px;
+  border: 1px solid #dadddc;
 }
 
-.list_item.expanded .list_detail
-{
-  display: block;
+input[type=checkbox]:checked +label {
+  background: #CC3846;
+  border-radius: 4px;
+  color: #33333f;
+  border: none;
+  vertical-align: top;
+  font-size: 12px;
+  height: 25px;
+  margin-right: 8px;
+  border: 1px solid #dadddc;
 }
 
-.list_expand
-{
-  float: right;
-  cursor: default;
-  background: url('expand20.png') 0 0 no-repeat;
-  width: 20px;
-  height: 20px;
-  margin: 2px 4px 0 0;
+@keyframes growDown {
+  0% {
+      transform: scaleY(0)
+  }
+  100% {
+      transform: scaleY(1)
+  }
 }
 
-.list_help
+.list_item.kbd_list_item.expanded .list_detail
 {
-  float: right;
-  cursor: default;
-  background: url('help20.png') 0 0 no-repeat;
-  width: 20px;
-  height: 20px;
-  margin: 2px 4px 0 0;
+  display: block;
+ /* animation: growDown 500ms ease-in-out forwards;
+  transform-origin: top center;*/

```

Comment:
If you uncomment these lines.  You well see nice animation on the keyboard layout expanding. However it then breaks the pop-up which use postion: fixed.

Reply:
I'm not all that keen on animations -- they slow down the user experience :grin:

Reply:
Ha true can be or flair and no function,  this wasn't that slow an animation it sort of made the flow less harsh and focused the eyesto the correct spot. However, it isn't worth the effort. If it breaks the pop-up